src/main/java/lcduke/Ui.java

```diff
@@ -0,0 +1,68 @@
+package lcduke;
+
+import java.util.Scanner;
+
+/** Ths creates an Ui object.
+ */
+
+public class Ui {
+
+    /** This is the constructor to create the Ui Object.
+     */
+    public Ui() {
+        System.out.println("    ____________________________________________________________");
+        System.out.println("     Hello! I'm Duke");
+        System.out.println("     What can I do for you?");
+        System.out.println("    ____________________________________________________________\n");
+    }
+
+    public void showLoadingError(){
```

Comment:
You should complete this method before openning a pull request or add a TODO comment to let others know that this method is unfinished.